### DISCUSSION TITLE
Problem in firefox: carousel and fancybox

### DISCUSSION THREAD

**Discussion by ghost** [fancyapps/ui#643] [Q&A]
When going to full screen in firefox from certain initial widths, the demo at [this link](https://stackblitz.com/edit/js-2k1udh?file=index.html,index.js,style.css) does not work right.
The images are not centered correctly, and the thumbnails aren't present.
This problem is not in Chrome.
Any ideas on how to fix?
![img_1](https://github.com/fancyapps/ui/assets/77803085/d938cf3e-0a6f-4392-ab65-071a3a051ba9)

**Comment by fancyapps** [comment]
Hi,

Thanks for reporting, I will fix this issue.

**Reply by fancyapps** [reply]
@cw-code-1 It seems that Firefox needs this CSS snippet:

```css
.fancybox__carousel {
  min-width: 0;
  min-height: 0;
}
```

**Reply by ghost** [reply]
Thanks, it seems that only min-width:0; is needed.

As a side note, it would be helpful to have versions of the more complicated examples that don't use modules in the js.
Long story short, I (and probaly many others) don't know how to implement modules in wordpress.
And since I'm making a small personal site, I just want an quick easy way to use the nice examples.
I was able to do what I wanted pretty easily, using the browser side examples and the same idea as [this example.](https://codepen.io/mohan-aiyer/pen/bGxrpoN)

It would be great if your more complicated examples also had versions that don't use modules.

Having played around with slicker, swiper and some others, I have to say fancybox with the carousel is very easy to setup and it's functionality is outstanding.
Many thanks for making this available.
